components/calendar/src/lib.rs

```diff
@@ -43,34 +43,6 @@
 //! // Answering questions about days in month and year.
 //! assert_eq!(date_iso.days_in_year(), 366);
 //! assert_eq!(date_iso.days_in_month(), 30);
-//!
```

Comment:
Suggestion (optional): if you want to save these docs you could move them onto DateDuration itself